pages/Teams/team-profile-page.cfm

```diff
@@ -54,6 +54,41 @@
   GROUP BY lastName, firstName
 </cfquery>
 
+<cfquery name="getTeamDataM" datasource="roundleague"> 
```

Comment:
"getTeamDataM" change to something that's easier to understand

Reply:
thanks for the catch dude!